cmd/cert-checker/main.go

```diff
@@ -238,10 +239,45 @@ var expectedExtensionContent = map[string][]byte{
 	"1.3.6.1.5.5.7.1.24": {0x30, 0x03, 0x02, 0x01, 0x05}, // Must staple feature
 }
 
+// checkValidations checks the database for matching authorizations that were
+// likely valid at the time the certificate was issued. Authorizations with
+// status = "deactivated" are counted for this, so long as their validatedAt
+// is before the issuance and expiration is after.
+func (c *certChecker) checkValidations(cert core.Certificate, dnsNames []string) error {
+	authzs, err := sa.SelectAuthzsMatchingIssuance(c.dbMap, cert.RegistrationID, cert.Issued, dnsNames)
+	if err != nil {
+		return fmt.Errorf("error checking authzs for certificate %s: %w", cert.Serial, err)
+	}
+
+	if len(authzs) == 0 {
+		return fmt.Errorf("no relevant authzs found valid at %s",
+			cert.Issued)
```

Comment:
nit: unnecessary line break
```suggestion
		return fmt.Errorf("no relevant authzs found valid at %s", cert.Issued)
```